models/countries.go

```diff
@@ -0,0 +1,138 @@
+package models
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+const COUNTRIES_API = "https://restcountries.eu/rest/v2/alpha/"
```

Comment:
By convention, the variables, constants, structs, etc, are named using CamelCase

Reply:
Also, all the exported things  (constants, variables, functions, structs, etc) should have a description